ajax/libs/gibberish-aes/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gibberish-aes",
+  "filename": "gibberish-aes.min.js",
```

Comment:
@iwdmb 
I saw the filename in the `dist` is `gibberish-aes-1.0.0.min.js`.
If we want to use auto-update, the filename couldn't with version number.
Would you ask the author to change the main filename without version number? Thanks.